### PR TITLE
Feature/buyer address

### PR DESCRIPTION
**Pull request by JChartier001**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: merch-dropper/src/utils/axiosWithAuth.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ export const axiosWithAuth = () => {
   const token = localStorage.getItem("token");
 
   return axios.create({
-    baseURL: "https://merchdropper-production.herokuapp.com",
```
Comment (killinq-joke) [inline merch-dropper/src/utils/axiosWithAuth.js:14]: look here

Comment (landoDev) [reply merch-dropper/src/utils/axiosWithAuth.js:14]: We want it to go to merch-dropper.herokuapp.com now. Do I have that right?

File: merch-dropper/src/components/Dashboard/InventoryList.js [hunk 2]
```diff
@@ -30,7 +30,8 @@ function InventoryList({ history }) {
       setStores(resStore.data);
       const storeID = resStore.data.id;
       const resProducts = await axios.get(
-        `https://merchdropper-production.herokuapp.com/api/products/store/${storeID}`
+        `http://localhost:4000/api/products/store/${storeID}`
```
Comment (landoDev) [inline merch-dropper/src/components/Dashboard/InventoryList.js:33]: I think this was mentioned just putting it here to be extra sure

File: merch-dropper/src/components/Shirt/DesignShirt.js [hunk 3]
```diff
@@ -23,15 +23,23 @@ const DesignShirt = ({
...
- 
-  // const addProduct = function() {
-  //   AddProduct(garment);
-  // };
+  thumbRender,
+  product,
+  setProduct
+}) => {  
+  
+  // const [product, setProduct] = useState({
```
Comment (landoDev) [inline merch-dropper/src/components/Shirt/DesignShirt.js:31]: If we won't need this commented code we can toss it I think

**Pull request merged**